packages/themes/src/index.ts

```diff
@@ -1,5 +1,5 @@
 export * from './components/index.js';
 export * from './composables/index.js';
-export * from './schemas/index.js';
 export * from './stores/index.js';
+export * from './types/index.js';
```

Comment:
Do we need a new folder/file just for re-export? Re-exporting direct from the package should do
```suggestion
export { Theme } from '@directus/types';
```